compair/static/modules/user/user-module.js

```diff
@@ -347,6 +370,50 @@ module.controller("UserCourseController",
             }
         };
 
+        $scope.updateLTIList = function() {
+            UserLTIUsersResource.get({'id': $scope.userId}).$promise.then(
+                function(ret) {
+                    $scope.lti_users = ret.objects;
+                }
+            );
+        };
+        $scope.updateLTIList();
+
+        $scope.unlinkLTI = function(lti_user) {
+            var params = {
+                id: $scope.userId,
+                ltiUserId: lti_user.id,
+            }
+            UserLTIUsersResource.deleteById(params).$promise.then(
+                function (ret) {
+                    Toaster.success("LTI Unlink", "Successfully unlinked the LTI user as requested.");
+                    $scope.updateLTIList();
+                }
+            );
+        }
+
+        $scope.updateThirdPartyUserList = function() {
```

Comment:
better to pre-load this (make it conditional in preloader with `AuthTypesEnabled.cas`)